### DISCUSSION TITLE
Fluxstore multivendor need Auction Feature

### DISCUSSION THREAD

**Discussion by shahriar5653** [inspireui/support#5579] [New Feature/Ideas Requests]
Hi Dear sir Fluxstore app need to add an auction feature with your app it is a very important feature of an e-commerce app so if you integrate an auction feature it increases your sales more and more

**Comment by thoiruddin** [comment]
> Hi Dear sir Fluxstore app need to add an auction feature with your app it is a very important feature of an e-commerce app so if you integrate an auction feature it increases your sales more and more

Thanks for this one. Yes! We need this. It's take 3 years for me to wait this "Auction" things + notification to bidder and seller also chat with them after winning or automatically deduct bidder credit wallet or ask them to make payment with time limit before the seller can repost their product back. I saw lot's of marketplace product, not all have auction features. Hope fluxstore can consider this one and integrate with mobile application too.

**Comment by creationwebuk** [comment]
We need auction compatibility or the app needs to automatically hide formats it can't support!
